foodsaving/invitations/serializers.py

```diff
@@ -45,3 +47,20 @@ class InvitationAcceptSerializer(serializers.Serializer):
     def update(self, invitation, validated_data):
         invitation.accept(self.context['request'].user)
         return invitation
+
+
+class InvitationResendEmailSerializer(serializers.Serializer):
+    from datetime import timedelta
+
+    created_at = serializers.DateTimeField()
+
+    def validate(self, attrs):
+        if not timezone.now() >= attrs['created_at'] + self.timedelta(hours=1):
+            raise serializers.ValidationError(
+                _('An invitation has already been sent to this e-mail address, you can only resend after one hour!')
+            )
+        return attrs
+
+    def update(self, invitation, validated_data):
+        invitation.resend_invitation_email()
```

Comment:
This can be called directly in the api action method if the permission checks pass, the serializer is not needed then :)